src/components/data_comps/datm/cime_config/config_component.xml

```diff
@@ -111,6 +113,8 @@
       <value compset="^SSP585_">SSP5-8.5</value>
       <value compset="^HIST"   >20tr</value>
       <value compset="^20TR"   >20tr</value>
+      <value compset="^OMIP_DATM%IAF.*_POP2%[^_]*ECO">omip</value>
+      <value compset="^OMIP_DATM%JRA.*_POP2%[^_]*ECO">omip</value>
```

Comment:
I'm using `cesm2_omip` for `DATM_PRESAERO` because the forcing is from a CESM2 coupled run, and it has been extended in time, back into the 1600's and forward to 2018, to be compatible with OMIP protocols (repeat 1850 values prior to 1850).

I'm using `omip` for `DATM_CO2_TSERIES` because it has similarly been extended in time (repeat 1850 values prior to 1850). I didn't use `cesm2_omip` here because the forcing data is obs based, it is not derived from a CESM2 run.

That's my motivation for using different labels.

That said, I'm open to suggestions for better names. I'm all ears for a better way to signify 'repeat 1850 values prior to 1850'.

Reply:
@jedwards4b , is there a place in CIME to document the intended meaning of these labels: `cesm2_omip` for `DATM_PRESAERO` and `omip` for `DATM_CO2_TSERIES`?